module12/src/main/java/com/TEKcamp/module12/Entities/User.java

```diff
@@ -0,0 +1,73 @@
+package com.TEKcamp.module12.Entities;
+
+import javax.persistence.*;
+
+@Entity
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @Column (nullable = false, unique = true)
+    private String userId;
```

Comment:
nullable, and unique userID, but you didn't implement any sort of generation strategy for this in your service layer. The way you have your app set up, the UserID has to be provided by the user, which is not a great strategy.